### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matthewattanasio**
## Requirements

 * [ ] This is Software as a Service not self hosted
 * [ ] It has a free tier not just a free trial
 * [ ] Pricing information is clearly visible without signup or phone calls
 * [ ] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1915,6 +1915,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [OpenUtils](https://openutils.org/) - There are various free tools available for developers, such as HTML/CSS/JavaScript formatters, minifiers, converters, encoders/decoder,s and many others.
   * [SimpleRestore](https://simplerestore.io) - Hassle-free MySQL backup restoration. Restore MySQL backups to any remote database without code or a server.
   * [360Converter](https://www.360converter.com/) - Free tier useful website to convert: Video to Text && Audio to Text && Speech to Text && Real-time Audio to Text && YouTube Video to Text && add Video Subtitle. Maybe it will be helpful in a short video conversion or in a short youtube tutorial:)
+  * [Sign Customiser](https://www.signcustomiser.com/) - Free to setup on your store - A Shopify app that allows merchants selling signage products to sell custom signs on their store.
```
Comment (ripienaar) [inline README.md:1918]: This doesnt appear to have a free tier, what exactly is free?

**Comment by madhuryadutta** [comment]
@ripienaar ,
Their apps page in shopify says **"Completely free for all development stores"** ,does it mean **Not for production** use. Sorry, I have no idea about shopify service ).
@matthewattanasio correct me if I'm wrong .

Pricing mention on the **App Page**
-

Screenshot from 2024-01-31 19-03-50

**Shopify App Page**

-

image

**Comment by ripienaar** [comment]
its not required to be free for production to be on the list

**Comment by madhuryadutta** [comment]
> its not required to be free for production to be on the list

Okay , Thanks for the clarification, @ripienaar .Then this service should be on the list . 👍👍

**Comment by matthewattanasio** [comment]
Thanks, guys, it sounds like you have worked out my app's free structure :)

Looking forward to being on the list.

**Comment by ripienaar** [comment]
Thanks, in the end though I dont think I'll add this to the list.  It just doesnt fit here.  I am sure there are better places to list it.

```
The scope of this particular list is limited to things that infrastructure developers (System Administrator, DevOps Practitioners, etc.) are likely to find useful. We love all the free services out there, but it would be good to keep it on topic. It's a grey line sometimes, so this is opinionated; please don't feel offended if I don't accept your contribution.
```

**Pull request closed without merging**